dts/bindings/mtd/cypress,fm25w256.yaml

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2021 Caspar Friedrich <c.s.w.friedrich@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+description: 256-Kbit (32K × 8) Serial (SPI) F-RAM
+
+compatible: "cypress,fm25w256"
+
+include: [eeprom-base.yaml, spi-device.yaml]
+
+properties:
+  wp-gpios:
+    type: phandle-array
+    required: false
+    description: |
+      Enable write protection on status register, active low.
+      The functionality is not supported by the driver. If the pin is set, the
+      driver ensures that the GPIO is configured correctly to not interfere with
+      normal operation.
+
+  hold-gpios:
+    type: phandle-array
+    required: false
+    description: |
+      Suspend the current spi operation, active low.
+      The functionality is not supported by the driver. If the pin is set, the
+      driver ensures that the GPIO is configured correctly to not interfere with
+      normal operation.
```

Comment:
I don't see how we would ever incorporate support for this into the SPI stack (put an ongoing SPI transaction on hold, use the SPI bus for something else, then resume the previous transaction). I suggest leaving out the HOLD pin completely.

If a boards needs to drive the HOLD pin to a static level during boot, it can do so in board initialisation code.

Reply:
Fair. Will remove HOLD.